src/main/java/com/aca/exam/exam3/service/core/UserPlateNumberService.java

```diff
@@ -0,0 +1,16 @@
+package com.aca.exam.exam3.service.core;
+
+import com.aca.exam.exam3.entity.UserPlateNumber;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface UserPlateNumberService {
+    UserPlateNumber create(UserPlateNumberCreationParams params);
+
+    UserPlateNumber getByPlateNumber(String plateNumber);
+
+    List<UserPlateNumber> getAllUserPlateNumbersByUser(String passport);
+
+    UserPlateNumber update(Long id, UserPlateNumberUpdateParams params);
```

Comment:
it is not clear whom id is this. if it is user's id please write userId, if userPlateNUmber's id then you already have it in update params. I suggest to have everything in update params parameter object